lib/jekyll/assets/plugins/html/pic.rb

```diff
@@ -35,8 +35,8 @@ def run
                 @args[:srcset].each do |v|
                   args = "#{@args[:argv1]} @srcset #{v}"
                   p << Tag.new("asset", args, bctx).render(ctx)
-                  p.img @args.to_h(html: true)
                 end
+                p.img @args.to_h(html: true)
```

Comment:
This needs to be spaced between the end, and should be bracketed if on it's own line.

Reply:
> should be bracketed if on it's own line

Sorry if this is pretty obvious, but I'm not at all familiar with Ruby's syntax: what do you mean by that?